scrapers/base.py

```diff
@@ -5,6 +5,9 @@ class BaseScraper(object):
     """
     Base scraper class to standardize the main scraper functionality
     """
+    teiid_hostname = 'virtualdb.engineering.redhat.com'
```

Comment:
Can we just have this as teiid_host instead of teiid_hostname ? Just for uniformity with the teiid class

Reply:
Sure